### ISSUE TITLE
<br> in Mermaid breaks generated SVG

### ISSUE THREAD

**Issue by cowboy** [yuzutech/kroki#924]
How to reproduce:

1. Visit https://kroki.io/
2. Select "Mermaid"
3. Change text content to the following:

```mermaid
graph TD
  A{{ Hello<br>World }}
```

4. See preview update correctly
![image](https://user-images.githubusercontent.com/54051/135148543-e5280e59-3cc0-4093-a062-369d02451434.png)
5. Click "Copy" to copy SVG URL to clipboard (https://kroki.io/mermaid/svg/eNpLL0osyFAIceFSUHCsrlbwSM3JybdJKrILzy_KSVGoreUCALUTCto=)
6. Open URL and see big red error
![image](https://user-images.githubusercontent.com/54051/135148716-9f07167a-6c70-4b07-a8e8-815b8b1eb542.png)

**Comment by Mogztter** [comment]
Thanks for the detailed report 🙌🏻 

I believe that the correct syntax should be:

```
graph TD
  A{{ Hello<br/>World }}
```

But interestingly, Kroki returns the following SVG:

```xml
<svg id="mermaid-1632857169981" width="112.859375" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" height="80" viewBox="0 0 112.859375 80">
  <style>#mermaid-1632857169981{font-family:"trebuchet ms",verdana,arial,sans-serif;font-size:16px;fill:#333;}#mermaid-1632857169981 .error-icon{fill:#552222;}#mermaid-1632857169981 .error-text{fill:#552222;stroke:#552222;}#mermaid-1632857169981 .edge-thickness-normal{stroke-width:2px;}#mermaid-1632857169981 .edge-thickness-thick{stroke-width:3.5px;}#mermaid-1632857169981 .edge-pattern-solid{stroke-dasharray:0;}#mermaid-1632857169981 .edge-pattern-dashed{stroke-dasharray:3;}#mermaid-1632857169981 .edge-pattern-dotted{stroke-dasharray:2;}#mermaid-1632857169981 .marker{fill:#333333;stroke:#333333;}#mermaid-1632857169981 .marker.cross{stroke:#333333;}#mermaid-1632857169981 svg{font-family:"trebuchet ms",verdana,arial,sans-serif;font-size:16px;}#mermaid-1632857169981 .label{font-family:"trebuchet ms",verdana,arial,sans-serif;color:#333;}#mermaid-1632857169981 .cluster-label text{fill:#333;}#mermaid-1632857169981 .cluster-label span{color:#333;}#mermaid-1632857169981 .label text,#mermaid-1632857169981 span{fill:#333;color:#333;}#mermaid-1632857169981 .node rect,#mermaid-1632857169981 .node circle,#mermaid-1632857169981 .node ellipse,#mermaid-1632857169981 .node polygon,#mermaid-1632857169981 .node path{fill:#ECECFF;stroke:#9370DB;stroke-width:1px;}#mermaid-1632857169981 .node .label{text-align:center;}#mermaid-1632857169981 .node.clickable{cursor:pointer;}#mermaid-1632857169981 .arrowheadPath{fill:#333333;}#mermaid-1632857169981 .edgePath .path{stroke:#333333;stroke-width:1.5px;}#mermaid-1632857169981 .flowchart-link{stroke:#333333;fill:none;}#mermaid-1632857169981 .edgeLabel{background-color:#e8e8e8;text-align:center;}#mermaid-1632857169981 .edgeLabel rect{opacity:0.5;background-color:#e8e8e8;fill:#e8e8e8;}#mermaid-1632857169981 .cluster rect{fill:#ffffde;stroke:#aaaa33;stroke-width:1px;}#mermaid-1632857169981 .cluster text{fill:#333;}#mermaid-1632857169981 .cluster span{color:#333;}#mermaid-1632857169981 div.mermaidTooltip{position:absolute;text-align:center;max-width:200px;padding:2px;font-family:"trebuchet ms",verdana,arial,sans-serif;font-size:12px;background:hsl(80,100%,96.2745098039%);border:1px solid #aaaa33;border-radius:2px;pointer-events:none;z-index:100;}#mermaid-1632857169981:root{--mermaid-font-family:"trebuchet ms",verdana,arial,sans-serif;}
  </style>
  <g>
    <g class="output">
      <g class="clusters"></g>
      <g class="edgePaths"></g>
      <g class="edgeLabels"></g>
      <g class="nodes">
        <g class="node default" id="flowchart-A-1" transform="translate(56.4296875,40)" style="opacity: 1;"> 
          <polygon points="16,0 80.859375,0 96.859375,-32 80.859375,-64 16,-64 0,-32" transform="translate(-48.4296875,32)" class="label-container"></polygon>
          <g class="label" transform="translate(0,0)">
            <g transform="translate(-22.4296875,-22)">
              <foreignObject width="44.859375" height="44">
                <div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; white-space: nowrap;">Hello<br>World</div>
              </foreignObject>
            </g>
          </g>
        </g>
      </g>
    </g>
  </g>
</svg>
```

As you can see the `<div>` element inside the `<foreignObject>` element contains `Hello<br>World` (instead of `Hello<br>World`).

I think that's the reason why we get a parsing error from the browser.
I can reproduce this issue using Chrome or Firefox.

**Comment by cowboy** [comment]
Yeah, I saw the same thing with both `<br>` and `<br/>`

**Comment by Mogztter** [comment]
I found a few issues on about line breaks and `<br>` on https://github.com/mermaid-js/mermaid most are resolved though.

I will take a closer look and try to fix this issue for the next release (0.15.0).

**Comment by cowboy** [comment]
FWIW, this code

```mermaid
graph TD
  A{{ Hello<br/>World }}
```

Generates this SVG embed URL

https://kroki.io/mermaid/svg/eNpLL0osyFAIceFSUHCsrlbwSM3JybdJKtK3C88vyklRqK0FALN_Cv8=

Which should work after the release
